lib/src/util/class_names.dart

```diff
@@ -150,6 +166,32 @@ class ClassNameBuilder {
     _blacklistBuffer.write(className);
   }
 
+
+
+  /// Merges together the classes and blacklists present in another `ClassNameBuilder` instance.
+  /// 
+  /// Takes [other._blacklistBuffer] and merges it into [_blacklistBuffer]
+  /// and takes [other._classNamesBuffer] and merges it into [_classNamesBuffer].
```

Comment:
This second section is redundant, since it's just describing the below code; we could probably just remove it.